modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -225,31 +227,47 @@ export function isValidAmount(amount: string): boolean {
  * @param {string} data The wallet creation data to decode
  * @returns {string[]} - The list of signer addresses
  */
-export function decodeWalletCreationData(data: string): string[] {
-  if (!(data.startsWith(walletInitializationFirstBytes) || data.startsWith(v1WalletInitializationFirstBytes))) {
+export function decodeWalletCreationData(data: string): WalletInitializationData {
```

Comment:
This changes the exported return. What is the possible disruption as a result for internal and external users? Does this need a BREAKING CHANGE in the footer and a `major` bump to the package?

Reply:
I'm leaning this direction due to this portion of the definition of a breaking change: `Removal of an allowed parameter, request field or response field`

Reply:
@mmcshinsky-bitgo This is a great note! In the case of `decodeWalletCreationData`, this method is not typically exposed directly (externally), and is only used for ETH tx building. The method is also not used in BGMS anywhere which gives us additional confidence about how seldom it is used. That being said if it is policy that any return type change constitutes a major bump then we can go through those proceedings

Reply:
Yeah, it's that weird spot since this is an exported function, we would have to support the possibility that there are external users who could be using it even if it doesn't make sense to us or our implementations already take care of it.

Reply:
Ok Got it. So I think because of this possibility that external users might be using this method, I should add Breaking Change in the commit footer, is that correct?

Reply:
Yeah, thanks :)

Reply:
done